packages/common/container-definitions/api-report/container-definitions.api.md

```diff
@@ -75,7 +75,7 @@ export interface ContainerWarning extends IErrorBase_2 {
 export interface IAudience extends IEventProvider<IAudienceEvents> {
     getMember(clientId: string): IClient | undefined;
     getMembers(): Map<string, IClient>;
-    getSelf: () => ISelf | undefined;
+    getSelf(): ISelf | undefined;
```

Comment:
when reviewing public APIs, I spotted this. I don't think the intention in this API was to have a mutable callback users could reassign OR call without the Audience object as its `this`. At least one implementation implemented this with a method, so I made the API match.